ps2/Makefile

```diff
@@ -0,0 +1,69 @@
+DEBUG              = 0
+MUTE_WARNINGS      = 1
+PS2_IP = 192.168.1.150
+
+TARGET = OpenSupaplex-debug.elf
+TARGET_RELEASE = OpenSupaplex.elf
+
+ifeq ($(DEBUG), 1)
+   OPTIMIZE_LV	:= -O0 -g
+	RARCH_DEFINES += -DDEBUG
+else
+   OPTIMIZE_LV	:= -O3
+	LDFLAGS :=  -s
+endif
+
+ifeq ($(MUTE_WARNINGS), 1)
+   DISABLE_WARNINGS := -Wno-sign-compare -Wno-unused -Wno-parentheses
+endif
+
+INCDIR = -I$(PS2DEV)/gsKit/include -I$(PS2SDK)/ports/include
+INCDIR += -Ips2/include -Ilibretro-common/include -Ideps -Ideps/stb -Ideps/7zip
+INCDIR += -Ideps/pthreads -Ideps/pthreads/platform/ps2 -Ideps/pthreads/platform/helper
+CFLAGS = $(OPTIMIZE_LV) $(DISABLE_WARNINGS) -ffast-math -fsingle-precision-constant
+ASFLAGS = $(CFLAGS)
+
+CFLAGS += -D__PS2__ -Dmain=SDL_main -DHAVE_SDL2
+
+LDFLAGS += -L$(PS2DEV)/gsKit/lib -L$(PS2SDK)/ports/lib -L.
+LIBS += -lSDL2main -lSDL2 -lSDL2_mixer -lpatches -lgskit -ldmakit -lps2_drivers -lxmp
+
+SRC = $(wildcard ../src/*.c) $(wildcard ../src/sdl_common/*.c) $(wildcard ../src/sdl2/*.c) ../src/null/virtualKeyboard.c ../src/lib/ini/ini.c
+EXCLUDE = ../src/sdl1/video.c
+SRC := $(filter-out $(EXCLUDE),$(SRC))
+OBJS = $(SRC:.c=.o)
+
+EE_OBJS += $(OBJS)
+EE_CFLAGS = $(CFLAGS)
+EE_CXXFLAGS = $(CFLAGS)
+EE_LDFLAGS = $(LDFLAGS)
+EE_LIBS = $(LIBS)
+EE_ASFLAGS = $(ASFLAGS)
+EE_INCS = $(INCDIR)
+EE_BIN = $(TARGET)
+EE_GPVAL = $(GPVAL)
+
+
+all: $(EE_BIN)
+
+clean:
+	rm -f $(EE_BIN) $(EE_OBJS)
+
+prepare:
+	ps2client -h $(PS2_IP) reset
+	ps2client -h $(PS2_IP) netdump
+
+run:
```

Comment:
As mentioned above, just like in https://github.com/sergiou87/open-supaplex/blob/e96fa69ad89de7aa155b5cfdf447f6b398f45462/ps3/Makefile#L139-L141
```suggestion
# Usage: PS2_IP="192.168.01.150" make run
run:
```